draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -648,6 +648,34 @@ Here, `eyJhbGci...emhlaUJhZzBZ` represents the SD-JWT and `eyJhb...dYALCGg` repr
 
 Other specifications or profiles of this specification may define alternative formats for transporting the Combined Format for Presentation that envelopes multiple such objects into one object, and provides Key Binding using means other than the Key Binding JWT.
 
+# JWS JSON Serialization {#json_serialization}
+
+(Work in progress...)
+
+This section describes an optional alternate format for SD-JWT using the JWS JSON Serialization from [@!RFC7515].
+
+For both the General and Flattened JSON Serialization, the SD-JWT is represented as a JSON object according
+to Section 7.2 of [@!RFC7515]. The disclosures (both for issuance and presentation) are included in the
```

Comment:
```suggestion
to Section 7.2 of [@!RFC7515]. The disclosures (both for issuance and presentation) MUST be included in the
```

Reply:
I was intentionally avoiding RFC 2119 key words like MUST here because it seemed unnecessary and potentially limiting.

Reply:
Do you have any specific use cases in mind where this might be limiting?

Reply:
A profile or something might want to base64url-encode the JWS JSON part and add the disclosures `~` separated to the end.  I don't like that. And defintily don't think this spec should describe it. But didn't want to preclude it.

Reply:
SHOULD this be a SHOULD then?

Reply:
let's keep as-is and see if the topic arises again.